app/src/main/java/com/futurice/freesound/utils/TimeScheduler.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2016 Futurice GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.futurice.freesound.utils;
+
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+import io.reactivex.Scheduler;
+import io.reactivex.functions.BiFunction;
+
+import static com.futurice.freesound.functional.Functions.apply;
+import static io.reactivex.schedulers.Schedulers.computation;
+
+/**
+ * Scheduler used for time based operations.
```

Comment:
I think this could use a little more documentation. Perhaps discuss the motivation of creating this class so newcomers know what problem is being solves and how to use it.